src/ui/common/src/components/notifications/NotificationsPopover.tsx

```diff
@@ -40,35 +31,16 @@ export const NotificationsPopover: React.FC<NotificationsPopoverProps> = ({
   const notificationsState = useSelector(
     (state: RootState) => state.notificationsReducer
   );
-  const [activeTab, setActiveTab] = useState(NotificationsTabs.All);
   const notifications: Notification[] = notificationsState.notifications;
 
-  const handleChangeTab = (event: React.SyntheticEvent, newValue: string) => {
-    event.preventDefault();
-    setActiveTab(NotificationsTabs[newValue]);
-  };
-
   const handleArchiveAll = (event: React.MouseEvent) => {
     event.preventDefault();
     dispatch(handleArchiveAllNotifications({ user, notifications }));
   };
 
   const filteredNotifications: Notification[] = notifications
     .filter((notification: Notification) => {
-      if (notification.level === NotificationLogLevel.Success) {
-        return false;
-      }
-      const association = notification.association.object;
-      if (activeTab === NotificationsTabs.All) {
-        return true;
-      } else if (activeTab === NotificationsTabs.Workflow) {
-        return (
-          association === NotificationAssociation.Workflow ||
-          association === NotificationAssociation.WorkflowDagResult
-        );
-      }
-
-      return true;
+      return notification.level !== NotificationLogLevel.Success;
```

Comment:
@vsreekanti This logic was there from before where we only show non-success notifications. Wanted to confirm that this what we want?

Reply:
Showing success notifications turned out to be a little too noisy, particularly when users had scheduled workflows, so I believe this is what we want.

Reply:
Yep, I think that's good for now. We can maybe make this an app setting in the future.